### ISSUE TITLE
Grammar schools rejected as private

### ISSUE THREAD

**Issue by rebecca-mccann** [yalla-coop/erica-playground#37]
### Device

Windows Laptop

### What browsers are you seeing the problem on?

Chrome

### What kind of issue is it?

Backlog / New idea

### Priority

Low

### Please describe your issue & include screenshots:

When you enter grammar schools, you are taken to the rejection page which tells you we don't support students from private schools - think it would be better to rephrase this as it will open us up to questions from grammar school students as they don't go to private school.

![image](https://github.com/yalla-coop/smf-app-QA-repo/assets/142805793/96e9350e-84f4-4c23-9dad-7743f4120a02)

Rebecca MC

### Page URL

https://app.socialmobility.org.uk/app-unsuccessful-school

**Comment by egasparini** [comment]
Thanks @rebecca-mccann!
@Wah-AuSMF this one is for you to review. Changing the copy on this page is surely an easy thing to do.
Let me know how you'd like the rephrase the sentence.
